specification/guestconfiguration/resource-manager/Microsoft.GuestConfiguration/preview/2018-06-30-preview/examples/getSoftwareUpdateProfile.json

```diff
@@ -0,0 +1,50 @@
+{
+  "parameters": {
+    "subscriptionId": "mySubscriptionId",
+    "resourceGroupName": "myResourceGroupName",
+    "vmName": "myVMName",
+    "api-version": "2018-06-30-preview"
+  },
+  "responses": {
+    "200": {
+      "headers": {},
+      "body": {
+        "id": "/subscriptions/mySubscriptionId/resourceGroups/myResourceGroupName/providers/Microsoft.Compute/virtualMachines/myVMName/providers/Microsoft.GuestConfiguration/softwareUpdateProfile/default",
```

Comment:
SoftwareUpdateProfile defined here: https://github.com/Azure/azure-rest-api-specs/pull/4950/files#diff-e53a4ef9e8ceed16b91a1997e37815e6R1017 does not have properties "id", "name", "type". Did you forget to inherit it from Resource?

Reply:
Inherited from Proxy resource now

Reply:
Inherited from proxy now